### DISCUSSION TITLE
Is podman python SDK runtime depend on podman ?

### DISCUSSION THREAD

**Discussion by Matiz7** [containers/podman#22195] [General]
in centos8  podman only upgrade to version 3.3.1 by yum .
I want latest version of podman.
I wonder whether or not podman python SDK runtime depend on podman 
if not , I can simplely use pip to install latest version of podman SDK instead complicated install process by building from source

**Comment by rhatdan** [comment]
Yes, can you update to centos 9?

**Reply by Matiz7** [reply]
I had use docker resolved it
